VSharp.ML.GameServer.Runner/Main.fs

```diff
@@ -328,7 +472,61 @@ let main args =
         with e ->
             printfn $"Failed on port {port}"
             printfn $"{e.Message}"
+    | Mode.SendModel ->
+        let model =
+            match args.TryGetResult <@ Model @> with
```

Comment:
Try to use ```GetResult``` with default value. Like [here](https://github.com/gsvgit/ImageProcessing/blob/943f9d0d12b37d3c4a3db078369bb0dc29535332/src/ImageProcessing/Main.fs#L35C33-L35C42)